app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
```

Comment:
- [ ] I can see your user's are displaying as expected, But your posts are not modified as expected. Kindly make sure user's all posts must be displayed.

Reply:
index, and show both must be equally updated